bot/seasons/evergreen/TriviaQuiz.py

```diff
@@ -0,0 +1,263 @@
+import asyncio
+import logging
+import random
+from dataclasses import dataclass
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+from fuzzywuzzy import fuzz
+
+from bot.constants import Roles
+
+
+logger = logging.getLogger(__name__)
+
+
+annoyed_expressions = ["-_-", "-.-"]
+
+wrong_ans_responses = [
+    "No one gave the correct answer",
+    "Losers",
+    "You guys really need to learn"
+]
+
+
+@dataclass
+class GameData:
+    """A dataclass for game data."""
+
+    owner: int
+    players = []
+    points = []
+    done_questions = []
+    category: str
+    question: dict = None
+    hints = 0
+    unanswered_questions = 0
+
+
+class TriviaQuiz(commands.Cog):
+    """A cog for all quiz commands."""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.questions = self.load_questions()
+        self.games = {}  # channel as key and value as instinct of dataclass GameData
+        self.categories = {
+            "retro": "Questions related to retro gaming."
+        }
+        self.inactivity_limit = 3  # Number of questions unanswered in a row after which quiz stops.
+
+    @staticmethod
+    def load_questions():
+        """Load the questions from  json file."""
+        p = Path("bot", "resources", "evergreen", "trivia_quiz.json ")
+        with p.open() as json_data:
+            questions = load(json_data)
+            return questions
+
+    @commands.group(name="tquiz", invoke_without_command=False)
+    async def tquiz(self, ctx):
+        """Trivia Quiz game for fun!"""
+        await ctx.send_help("tquiz")
+
+    @tquiz.command(name="start")
+    async def start(self, ctx, category=None):
+        """Start a quiz!
```

Comment:
From what I can tell, the prevailing style is to have the summary line be below the triple quotes for multiline docstrings.
```suggestion
        """
        Start a quiz!
```

Reply:
Correct. This is supposed to be linted but we've identified an issue with our linting configuration that's causing this check to be skipped, should have it fixed within the next day or two.